mappings/net/minecraft/nbt/NbtIo.mapping

```diff
@@ -1,10 +1,18 @@
 CLASS net/minecraft/unmapped/C_plrkqquc net/minecraft/nbt/NbtIo
+	FIELD f_zwmbytoi WRITE_FILE_OPEN_OPRTIONS [Ljava/nio/file/OpenOption;
+	METHOD m_cvfqsgxo createDataOutputStream (Ljava/io/OutputStream;)Ljava/io/DataOutputStream;
+	METHOD m_hjflxsyo createDataInputStream (Ljava/io/InputStream;)Ljava/io/DataInputStream;
 	METHOD m_hsrlntfm write (Lnet/minecraft/unmapped/C_oivssbvb;Ljava/io/DataOutput;)V
 		ARG 0 nbt
 		ARG 1 output
+	METHOD m_jsabavmz readCompound (Ljava/nio/file/Path;Lnet/minecraft/unmapped/C_fozrrtcx;)Lnet/minecraft/unmapped/C_hhlwcnih;
+	METHOD m_kfuaemox writeCompressed2 (Lnet/minecraft/unmapped/C_hhlwcnih;Ljava/nio/file/Path;)V
```

Comment:
there's got to be a name that doesn't have the `2`
mojmap calls this one `writeCompressed` and calls the method this would conflict with `write`

Reply:
Looking closer, the current `writeCompressed(NbtCompound compound, Path path)` goes through `writeCompressed(NbtCompound compound, OutputStream stream)` which creates its `DataOutputStream` via `createDataOutputStream` which uses `GZIPOutputStream`

the current `writeCompressed2(NbtCompound compound, Path path)` creates its `DataOutputStream` directly, without `GZIPOutputStream`

so I think `writeCompressed2` should just be `write`